examples/siro_sandbox/sandbox_app.py

```diff
@@ -65,10 +67,12 @@ def __init__(self, args, config, gui_input):
             config.habitat.simulator.habitat_sim_v0.enable_gfx_replay_save = (
                 True
             )
+            config.habitat.simulator.concur_render = False
```

Comment:
Without this I get `ESP_CHECK failed: Renderer was not created with a background render thread, cannot do async drawing` error.

Reply:
Seems like a fine workaround for now. We can investigate later if concurrent rendering (background-thread rendering) will help the sandbox app runtime perf.